packages/e2e-tests/specs/embedding.test.js

```diff
@@ -192,4 +192,34 @@ describe( 'Embedding content', () => {
 		await clickButton( 'Try again' );
 		await page.waitForSelector( 'figure.wp-block-embed-twitter' );
 	} );
+
+	it( 'should transform from video to embed block when YouTube URL is pasted', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( '/video' );
+		await page.keyboard.press( 'Enter' );
+		await page.click( '.editor-media-placeholder__url-input-container button' );
+		await page.keyboard.type( 'https://www.youtube.com/watch?v=DQuhA5ZCV9M' );
```

Comment:
These URLs are different than the ones above which are mocked. Do they need to be mocked to avoid a full network request?

Reply:
I could try to mock these too, but the situation is a bit different b/c the embed is created in a second step and I am not sure that image/audio/video blocks support setUpResponseMocking, so I may also need to add something similar to e2e-test-utils/src/mocks/create-embedding-matcher beforehand.

Reply:
added new mocked URLs!